README.md

```diff
@@ -33,6 +33,20 @@ Setup a Longclaw project
   $ longclaw start my_project
 ```
 
+Go to project directory and create missing migrations 
+
+```bash
+  $ python manage.py makemigrations home
+  $ python manage.py makemigrations catalog
+```
+
+Do migrations for whole project and run 
+
+```bash
+  $ python manage.py migrate
+  $ python manage.py runserver
```

Comment:
```suggestion
  $ python manage.py loadcountries
  $ python manage.py createsuperuser
  $ python manage.py runserver
```
Suggest adding commands to load the country data (for shipping module) and create an admin user